src/DynamicNLPModels.jl

```diff
@@ -157,27 +190,25 @@ end
 """
     LQDynamicModel(dnlp::LQDynamicData; condense=false)      -> LQdynamicModel
     LQDynamicModel(s0, A, B, Q, R, N; condense = false, ...) -> LQDynamicModel
-
 A constructor for building a `LQDynamicModel <: QuadraticModels.AbstractQuadraticModel` from `LQDynamicData`
-
 Input data is for the problem of the form 
-
 ```math
     minimize    \\frac{1}{2} \\sum_{i = 0}^{N-1}(s_i^T Q s_i + u_i^T R u_i) + \\frac{1}{2} s_N^T Qf s_N
     subject to  s_{i+1} = A s_i + B u_i  for i=0, 1, ..., N-1
+                gl \\le E s_i + F u_i \\le gu for i = 0, 1, ..., N-1            
                 sl \\le s \\le su
                 ul \\le u \\le uu
                 s_0 = s0
 ```
 ---
 
 If `condense=false`, data is converted to the form 
+
 ```math
     minimize    \\frac{1}{2} z^T H z 
-    subject to  0 \\le Jz \\le 0
+    subject to  lcon \\le Jz \\le ucon
```

Comment:
Ok for now. But we may want to update the docstring later for a better display in LaTeX